python/overrides.js

```diff
@@ -97,4 +97,12 @@
     'should send dom of version 11': {skipEmit: true},
     'check region by selector within shadow dom with vg': {skipEmit: true},
   	'check region by element within shadow dom with vg': {skipEmit: true},
+    'pageCoverage data is correct': {skipEmit: true},
+    'pageCoverage data is correct with vg': {skipEmit: true},
+    'should abort after close': {skipEmit: true},
+    'should abort unclosed tests': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
    "should abort unclosed tests": {skipEmit: true},
```